### ISSUE TITLE
outstanding RT issues

### ISSUE THREAD

**Issue by karenetheridge** [Perl-Toolchain-Gang/ExtUtils-Manifest#1]
These aren't linked via the resources bugtracker (that link goes to the main perl5 queue, which is correct since this dist is core-first).

However, there are issues outstanding at https://rt.cpan.org/Dist/Display.html?Queue=ExtUtils-Manifest

**Comment by mohawk2** [comment]
Is this still a live issue, or should it be closed?

**Comment by karenetheridge** [comment]
> Is this still a live issue?

Yes, there are lots of open issues tracked here. They could perhaps be opened as separate issues, but until then, this should stay open.

**Comment by dolmen** [comment]
I just closed [RT#21283](https://rt.cpan.org/Ticket/Display.html?id=21283), [RT#57586](https://rt.cpan.org/Ticket/Display.html?id=57586) and [RT#57046](https://rt.cpan.org/Ticket/Display.html?id=57046) as they have been fixed since 1.66. @bingos++

**Comment by karenetheridge** [comment]
@dolmen++

**Comment by mohawk2** [comment]
I have opened individual tickets here, for the remaining 4 open RTs:

| Subject | RT URL | Perl issue | EU:M GitHub issue |
|--|--|--|--|
| print to STDERR instead of warn in ExtUtils::Manifest | https://rt.cpan.org/Ticket/Display.html?id=21290 | https://github.com/Perl/perl5/issues/14005 | #41 |
| maniread() should return a tied, case-insensitive hash on VMS | https://rt.cpan.org/Ticket/Display.html?id=2746 | https://github.com/Perl/perl5/issues/14006 | #42 |
| Aborting 'make manifest' can leave you with a broken MANIFEST | https://rt.cpan.org/Ticket/Display.html?id=3109 | https://github.com/Perl/perl5/issues/14007 | #43 |
| Default MANIFEST.SKIP too aggressive about exclusions | https://rt.cpan.org/Ticket/Display.html?id=57507 | https://github.com/Perl/perl5/issues/14004 | #44 |

@karenetheridge Would you therefore be happy for me to close this?
